apps/aecore/lib/aecore/naming/name_util.ex

```diff
@@ -28,32 +28,39 @@ defmodule Aecore.Naming.NameUtil do
   def normalize_name(name), do: name |> :idna.utf8_to_ascii() |> to_string()
 
   @spec namehash(String.t()) :: binary()
-  defp namehash(name) do
+  def namehash(name) do
```

Comment:
I assume only the normalized_namehash should be uses, thus this function was private